### DISCUSSION TITLE
Static Web App is Lanching the Template. Not the Edited of Mine How to Fix that...

### DISCUSSION THREAD

**Discussion by safnimj** [microsoft/SATechnicalOnboarding#66] [Questions]
Static Web App in Azure is Lanching the Template.

Not the Customized Edited of Mine How to Fix that...

I think I missed some steps 😔

**Comment by ManuSquall** [comment]
Hello @mjsafni , may you provide a sscreenshot of the website rendered?

**Comment by AnthonyByansi** [comment]
@mjsafni do you mind sharing a screenshot of the issue you are facing so we can help?
